README.md

```diff
@@ -43,6 +43,31 @@ For more options, see the
 
 ```java -jar evosuite.jar -help```
 
+### EvoSuite on Docker Hub
+
+EvoSuite has a container image available on [Docker Hub](https://hub.docker.com/r/evosuite/evosuite). You can get the container by either pulling the image:
+
+```docker pull evosuite/evosuite:<version>```
+
+or by manually building the image locally:
+
+```
+git clone github.com/EvoSuite/evosuite.git
+cd evosuite
+docker build -f Dockerfile.java8 . --tag evosuite/evosuite:latest-java-8
+docker build -f Dockerfile.java11 . --tage evosuite/evosuite:latest-java-11
```

Comment:
There's a typo in this line, I suppose this should be --tag

Reply:
Yes, indeed. Is fixed now